include/z64save.h

```diff
@@ -1401,7 +1418,11 @@ typedef enum {
 // Unconfirmed: "Postman has delivered priority mail"
 #define WEEKEVENTREG_89_08 PACK_WEEKEVENTREG_FLAG(89, 0x08)
 
-#define WEEKEVENTREG_89_10 PACK_WEEKEVENTREG_FLAG(89, 0x10)
+// Player tried but failed to defend Romani Ranch from the alien invasion on Night 1
+// This is only set if the player is actually in the ranch when the aliens reach the barn.
+// If the player isn't in the ranch when this happens, then this weekeventreg will remain unset.
+#define WEEKEVENTREG_FAILED_TO_DEFEND_AGAINST_ALIENS PACK_WEEKEVENTREG_FLAG(89, 0x10)
```

Comment:
This is not checked by any currently-decompiled code. And given the nature of the code that isn't currently decompiled, I don't believe it's checked by anything, probably because it doesn't exactly behave in an intuitive fashion.